package.json

```diff
@@ -33,7 +33,7 @@
     }
   },
   "dependencies": {
-    "@typescript-eslint/parser": "^6.7.5",
+    "@typescript-eslint/parser": "^7.14.1",
```

Comment:
That could be a breaking change?

Reply:
@JounQin I guess it shouldn't be. CI is failing because of outdated node version

_[lint]   36:5  error  "@typescript-eslint/parser@^7.14.1" is not compatible with "node@>=16.10.0". Allowed is: "node@^18.18.0||>=20.0.0"  node-dependencies/compat-engines_

Reply:
Then it's a breaking change as it changes required node engine.

Maybe more breaking features should be considered, but I don't have enough time these days.